### PR TITLE
add the list of available keys

### PR DESCRIPTION
**Pull request by keichwa**
@mcalmer do you think it makes sense to add such a list of GPG keys?

close https://github.com/SUSE/spacewalk/issues/10265

### CODE REVIEW

**Comment by Bischoff** [comment]
I would remove the list of keys, but add a table 

  version of sles es <-> key to use

File: modules/client-configuration/pages/clients-rh.adoc [hunk 1]
```diff
@@ -380,15 +380,26 @@ By default, {rhel} does not trust the GPG key for {productname} {centos} client
...
+
+----
+ptf-gpg-pubkey-b37b98a9.key
+res-gpg-pubkey-0182b964.key
+sle10-gpg-pubkey-9c800aca.key
+sle11-gpg-pubkey-307e3d54.key
+sle12-gpg-pubkey-39db7c82.key
+sle12-reserve-gpg-pubkey-50a3dd1c.key
+----
+
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-rh.adoc]: RES6-SUSE-Manager-Tools uses sle11-gpg-pubkey-307e3d54.key
RES7-SUSE-Manager-Tools uses sle12-gpg-pubkey-39db7c82.key
RES8-SUSE-Manager-Tools uses sle12-gpg-pubkey-39db7c82.key
Ubuntu-16.04-SUSE-Manager-Tools uses sle12-gpg-pubkey-39db7c82.key
Ubuntu-18.04-SUSE-Manager-Tools uses sle12-gpg-pubkey-39db7c82.key

**Comment by keichwa** [comment]
Thanks!  I hope I got it right.

**Comment by Bischoff** [comment]
Yes, it's fine now. Thanks Karl.

**Pull request merged**